wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+# Use this script to test if a given TCP host/port are available
+
```

Comment:
Implement wait_for_db [management command](https://docs.djangoproject.com/en/4.0/howto/custom-management-commands/), which waits for the database to be available. So your services won't throw any errors during the docker-compose up command;

Reply:
I don't understand this request, command already implemented, wait-for-it.sh script represent logic which waits for the database to be available and there won't be any errors

Reply:
read requirements for this task,  may help

Reply:
Implement wait_for_db [management command](https://docs.djangoproject.com/en/4.0/howto/custom-management-commands/), which waits for the database to be available

Implemented command wait-for-it using public solution script, , which waits for the database to be available

Reply:
Did you follow the link? This task is about django management commands and understanding how it works.